ethereum/contracts/Bridge.sol

```diff
@@ -127,19 +129,21 @@ contract Bridge is IBridge, AccessControl {
     }
 
     modifier verifySwapAmount(uint256 amount) {
-        // NOTE(pb): Commenting-out check against `swapFee` in order to spare gas for user's Tx, relying solely on check
-        //  against `swapMin` only, which is ensured to be `>= swapFee` (by `_setLimits(...)` function).
-        //require(amount > swapFee, "Amount must be higher than fee");
-        require(amount >= swapMin, "Amount bellow lower limit");
-        require(amount <= swapMax, "Amount exceeds upper limit");
+        require(amount >= swapMin, "Amount below swap min limit");
+        require(amount <= swapMax, "Amount exceeds swap max limit");
         _;
     }
 
-    modifier verifyReverseSwapAmount(uint256 amount) {
+    modifier verifyRefundAmount(uint256 amount) {
         require(amount <= swapMax, "Amount exceeds swap max limit");
         _;
     }
 
+    modifier verifyReverseSwapAmount(uint256 amount) {
+        require(amount <= reverseSwapMax, "Amount exceeds swap max limit");
```

Comment:
`verifyReverseSwapAmount` is supposed to check `reverseSwapMin` as well, not that it'll make it more secure